internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -250,6 +244,7 @@ func (ws *workflows) deactivateDrainedDeployment(ctx context.Context, deployment
 			return fmt.Errorf("deployments.get: %w", err)
 		}
 
+		// FIXME: use single query for this and move this to the db layer?
```

Comment:
Is this meant to be answered and fixed before merging this PR?

Reply:
no, since this PR isn't handling this. I only added FIXME since  I think it should be done differently. Should I change it to TODO?